src/consumeAPI.js

```diff
@@ -0,0 +1,53 @@
+function getScoresFromApi() {
+  const resp = fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/XR8Xsx8fKmg97cQOTiMP/scores')
+    .then((response) => response.json())
+    .then((json) => json);
+  return resp;
+}
```

Comment:
- [ ] you need to make use of the `arrow functions` instead of the `function` keyword
- [ ] you need to also use the `async` and `await`

for example
```
const getScoresFromApi = async () => {
  try {
    const response = await fetch(
      'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/XR8Xsx8fKmg97cQOTiMP/scores'
    )

    return response.json()
  } catch (error) {
    throw new Error('something went wrong')
  }
}
```

Refers to use this example above to see how to write es6 syntax of writing `functions` instead of using the declarative way for writing `functions`, you can see from the example above how to use the `async` and `await` for your `functions`

Please do this for everywhere you're using the `function` keyword

Reply:
thanks for suggestions, i implemented all